quantumarticle.cls

```diff
@@ -1353,11 +1371,17 @@ class for Quantum - the open journal for quantum science (http://quantum-journal
 	{%
 	}
 	{	
-		\ifnumequal{\pdfoutput}{0}
-		{%
-			\ClassError{quantumarticle}{Please add \string\pdfoutput=1 within the first 5 lines of the preamble of your main LaTeX file as suggested by the arXiv (https://arxiv.org/help/submit_tex). Without this, the arXiv compiles your document with latex instead of pdflatex and hyperlinks in the bibliography broken over several lines will not work properly. See http://tex.stackexchange.com/questions/182523/publishing-to-arxiv-with-hyperlinks for more information. If you still want to use standard latex (for example because you depend on psfrag) instead of pdflatex despite this problem, you can disable this error by giving the nopdfoutputerror option to quantumarticle. In this case please check carefully that the links in your document, especially in the bibliography work properly.}{}%see also http://tex.stackexchange.com/questions/341466/latex-macros-in-classwarning-or-classerror for why \string is necessary
+		\ifcsdef{pdfoutput}
```

Comment:
After writing https://github.com/quantum-journal/quantum-journal/issues/75#issuecomment-478358088 I am no longer quite sure whether the code here actually does what it should do: Shouldn't there also be an error if `pdfoutput` is not defined? The current code throws an error if `pdfoutput` is defined and equal to 0. Shouldn't it through an error whenever `pdfoutput` is not defined or not equal to 1? I.e.: 
```
\ifnumequal{\pdfoutput}{1}
{}
{\ClassError{quantumarticle}{Please add...}}
```

Reply:
This is exactly what the code does, plus an additional check for `nopdfoutputerror` (which is basically "i don't care about pdfoutput") around the whole block.